test/integration/connectors/conftest.py

```diff
@@ -26,3 +33,67 @@ def temp_dir() -> Generator[Path, None, None]:
         logger.info(f"Created temp dir '{temp_path}'")
         yield temp_path
         logger.info(f"Removing temp dir '{temp_path}'")
+
+
+@pytest.fixture
```

Comment:
Can we add these to the one drive test file since they're so specific to that test?

Reply:
Moved all fixtures to the test file!